src/context/TopBanner.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {PropsWithChildren, PropsWithoutRef} from 'react';
+import { useClientEffect } from "../utils/hookts"
+
+interface ITopBannerMessage {
+  text: string
+  from: number
+}
```

Comment:
Beside the textContent of the message we also store the time when the message was closed. this could be useful for later evaluation of messages, e.g. showing a warning every so and so weeks, etc.